deploy/kubernetes/console/templates/analyzers.yaml

```diff
@@ -63,6 +63,9 @@ spec:
       imagePullSecrets:
       - name: {{.Values.dockerRegistrySecret}}
       {{- end }}
+      {{- if and (eq (printf "%s" .Values.kube.auth) "rbac") (.Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1") }}
```

Comment:
Does this need to be gated... as we will always have the stratos service?

Reply:
Doesn't need to be because it always exists